lib/Conversion/TorchToMhlo/Linear.cpp

```diff
@@ -71,9 +71,11 @@ Value getPermutedTensor(PatternRewriter &rewriter, Operation *op, Value input,
   return result.getResult();
 }
 
-void castContractingDim(PatternRewriter &rewriter, Operation *op, Value &lhs,
-                        Value &rhs, int64_t lhsContractingDim,
-                        int64_t rhsContractingDim) {
+RankedTensorType castContractingDim(PatternRewriter &rewriter, Operation *op,
```

Comment:
add an uni test for this fix?

Reply:
Some file checks are added in [BladeDISC](https://github.com/alibaba/BladeDISC/pull/642)